Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -1764,16 +1764,35 @@ public function testRenderRelationElementToString()
                 if ($value == 'associated_property') {
                     return $default;
                 }
+            }));
+
+        $element = new FooToString();
```

Comment:
It 's the diff that is wrongly detecting this as an addition. The real added line is line 1786 (below)